src/components/ErrorBoundary.js

```diff
@@ -13,7 +13,8 @@ class ErrorBoundary extends React.Component {
   }
 
   componentDidCatch(error, info) {
-    logger.error('An error was catched in React ErrorBoundary, ', error)
+    logger.error(new Error('An error was catched in React ErrorBoundary. '))
+    logger.error(error)
```

Comment:
Could we have the composite log error message instead of two? This would result two error entries instead of one though only one error occurs.

Reply:
It's on purpose. 
After discussion with @taylrj @YuCJ , we compromise on this error logging convention.
We can try this version for now, and observe the occurrence of error logs.

Reply:
The reason I wrote two error logs here is because of error reporting.
If `error` is not an `Error` object, then the error reporting won't catch the error.
Instead of checking `error instanceof Error` for every `logger.error`,
I plan to write informative error message for logging (since `error` comes everywhere, we need to do lots of check to know what it is).

Reply:
Why and what do we want to have the compromise for?